Description: Manpage enhancement
 Explicitly remember on manpage that Debian does not bind to any address
 by default.
---
Origin: Dererk <dererk@debian.org> 
Bug-Debian: 575705
Last-Update: <2010-06-08>

Index: openntpd-3.9p1+debian/ntpd.conf.5
===================================================================
--- openntpd-3.9p1+debian.orig/ntpd.conf.5	2011-02-09 16:52:04.037958974 -0300
+++ openntpd-3.9p1+debian/ntpd.conf.5	2011-02-09 19:41:39.731354052 -0300
@@ -14,7 +14,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT
 .\" OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\"
-.Dd June 17, 2004
+.Dd June 8, 2010
 .Dt NTPD.CONF 5
 .Os
 .Sh NAME
@@ -34,9 +34,11 @@
 They are as follows:
 .Bl -tag -width Ds
 .It Ic listen on Ar address
+OpenNTPd has the ability to sync the local clock to remote NTP servers and, if this directive is specified, can act as NTP server itself, redistributing the local clock.
+.Pp
 Specify a local IP address or a hostname the
 .Xr ntpd 8
-daemon should listen on.
+daemon should listen on to enable remote clients synchronization.
 If it appears multiple times,
 .Xr ntpd 8
 will listen on each given address.
